go.mod

```diff
@@ -1,18 +1,18 @@
 module github.com/sivchari/containedctx
 
-go 1.17
+go 1.18
```

Comment:
```suggestion
go 1.22.1

toolchain go1.23.3
```

Reply:
I prefer to use 1.22.1 since the golangci-lint uses go1.22.1 in themself. this linter might be almost used via golangci-lint.